### ISSUE TITLE
The .cwl file is not working for Tabularray

### ISSUE THREAD

**Issue by MiltosTex** [texstudio-org/texstudio#2559]
### Environment
- TeXstudio: version 4.3.1
- Qt: qt6 
- OS: Windows 10
- TeX distribution: texlive 2022

### When open the environment of Tabularray which can be done by:
````````````````````````````````````````````````````````````````````````````
\begin{tblr}{options->here you can pass any option or options of the package}
content...
\end{tblr}
`````````````````````````````````````````````````````````````````````````````````````

When someone tries to pass some options like (vlines, hlines, colspec, rowspec, etc, these normally are written in .cwl file of Tabularray) and begin to write the first letter or after a comma, normally a pop up window opens with the commands, so a user can use it and copy the command from this window if can not remember it.

### Actual behavior
In version 4.2.3 this pop up window I mentioned before opens as desired, but in versions 4.3.0 and 4.3.1 never comes up. I have noticed that only for package Tabularray and not any other package.

### write the following simple code in an empty TeXstudio window:
``````````````````````````````````````````````````````````````````````````
\usepackage{tabularray}
\begin{tblr}{c}
	content...
\end{tblr}
````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
then delete letter c and try to write an `h` or `r` or `v`. Normally after anyone of these letters a pop-up window would come up with some choices to select, but it never opens in version 4.3.0 and 4.3.1, in version 4.2.3 it opens instantly. See pictures below: 
First picture from version 4.2.3. Second picture from version 4.3.1

![Screenshot 2022-08-28](https://user-images.githubusercontent.com/112310814/187092544-72d36274-7dc9-4bac-bd0a-16de606a8260.png)
![Screenshot 2022-08-28 215331](https://user-images.githubusercontent.com/112310814/187092547-676cb9ee-c32a-42b9-a109-2dd3fac6dbf2.png)

**Comment by sunderme** [comment]
alright, keyval completion does not work anymore.
It is, unfortunately , the same issue as stated before. 
txs needs to know that the option field contains the tabular definition. Currently it can't handle keyval completion for the same field.
I take that as a feature request

**Comment by sunderme** [comment]
implemented with 77b51002856c8e1f63f90451f66671cdd614d8e0
